test/fixtures/url-setter-tests.js

```diff
@@ -1620,6 +1620,33 @@ module.exports =
                 "href": "sc://example.net/%23",
                 "pathname": "/%23"
             }
+        },
+        {
+            "comment": "File URLs and (back)slashes",
```

Comment:
Can you update the hash of the link in the comment at the top of this file

```
/* WPT Refs:
   https://github.com/w3c/web-platform-tests/blob/e48dd15/url/setters_tests.json
   License: http://www.w3.org/Consortium/Legal/2008/04-testsuite-copyright.html
*/
```

This should be 3eff1bd now.